include/uv/win.h

```diff
@@ -384,7 +384,10 @@ typedef struct {
       ULONG_PTR result; /* overlapped.Internal is reused to hold the result */\
       HANDLE pipeHandle;                                                      \
       DWORD duplex_flags;                                                     \
-      WCHAR* name;                                                             \
+      union {                                                                 \
+        WCHAR* name;                                                          \
+        SOCKET uds_socket;                                                    \
```

Comment:
no, because we now use IOCP at connect, it seems the IOCP will overwrite (result + pipeHandle)'s memory region. Thus for not breaking ABI, we can only reuse the `name`, otherwise we need other workaround, like firstly set the socket to handle->handle and set to connect.pipeHandle later at connect request handler.

Reply:
Ah, good to know!

Reply:
Left comment here for better understanding.